readme.txt

```diff
@@ -1,6 +1,6 @@
 === WooPayments - Fully Integrated Solution Built and Supported by Woo ===
 Contributors: woocommerce, automattic
-Tags: payment gateway, payment, apple pay, credit card, google pay, woocommerce payments
+Tags: woocommerce payments, apple pay, credit card, google pay, payment, payment gateway
```

Comment:
@htdat should we add subscription as a keyword as well?

Reply:
@naman03malhotra - I am not the person to make this decision. This change is suggested by the marketer @tckasa. More details here p1697828260699169/1697173249.458689-slack-CGGCLBN58